src/main/java/core/basesyntax/strategy/OperationStrategy.java

```diff
@@ -0,0 +1,20 @@
+package core.basesyntax.strategy;
+
+import core.basesyntax.model.FruitTransaction;
+import java.util.HashMap;
+import java.util.Map;
+
+public class OperationStrategy {
+    private final Map<FruitTransaction.Operation, OperationHandler> map;
+
+    public OperationStrategy(OperationHandler... strategies) {
+        this.map = new HashMap<>();
+        for (OperationHandler strategy : strategies) {
+            map.put(strategy.getType(), strategy);
+        }
```

Comment:
same issue

strategy is a pattern or service that provides proper implementation depending on some input, in our case implementation of OperationHandler.

operationHandler shouldn't be named as strategy, it's result of strategy job

```suggestion
        for (OperationHandler handler : handlers) {
            map.put(handler.getType(), handler);
        }
```

Please recheck code for similar issues